### PR TITLE
add terminus_schedules entry point

### PR DESCRIPTION
**Pull request by benoit-bst**
expose **terminus_schedules** into the playground

### CODE REVIEW

**Comment by benoit-bst** [comment]
I don't think so. There is no new fields inside Json. 
Only reusing (like stop point, line, and so on...)

**Comment by benoit-bst** [comment]
if you say so, I believe you. I’ll dig the topic.
Thanks @TeXitoi :smile_cat:

**Comment by TeXitoi** [comment]
Maybe that's not as bad I thought: auto extended view might do a good job, but a proper summary should improve the ux a lot.

**Comment by benoit-bst** [comment]
I have updated the **terminus_schedules** layout like **stop_schedule**, as mentioned in meeting. Now, it looks like that

terminus_schedules_response

@TeXitoi thanks to reviewed it

File: js/autocomplete.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ autocomplete._collections = [
 autocomplete._additionalFeatures = [
     'departures', 'journeys', 'places_nearby', 'pt_objects', 'route_schedules',
     'stop_schedules', 'arrivals', 'isochrones', 'heat_maps', 'traffic_reports',
-    'line_reports', 'equipment_reports'
+    'line_reports', 'equipment_reports', 'terminus_schedules'
```
Comment (pbougue) [inline js/autocomplete.js:46]: Here you're gonna publicly expose the feature (which means maintaining future backward compatibility and so on I guess).
Is this on purpose, or should it stay hidden like status (then no autocomplete...) as it is not documented in doc.navitia.io ?

Comment (benoit-bst) [reply js/autocomplete.js:46]: Dunno for the doc. 
I only manage the playground part for this feature. @kadhikari ?

Comment (kadhikari) [reply js/autocomplete.js:46]: As decided, we don't doccument in doc.navitia.io but only for playground.

Comment (nonifier) [reply js/autocomplete.js:46]: I agree with @pbougue on this one. When features are hidden or not documented, we don't provide auto-completion for them (which is the case for `/terminus_schedules`).

Even though the playground is not the source of truth integration wise, it's better to keep some consistency with our API.

Comment (benoit-bst) [reply js/autocomplete.js:46]: The **PO** said, we merge this PR after the doc update

File: js/extended.js [hunk 2]
```diff
@@ -324,6 +308,30 @@ extended.make.stop_area_equipment = function(context, json) {
     return res;
 };
 
+extended.make.terminus_schedule = function(context, json) {
+    return extended.defaultStopSchedule(context, json);
```
Comment (TeXitoi) [inline js/extended.js:312]: You could just `return extended.make.stop_schedule(context, json)` but whatever.

**Comment by benoit-bst** [comment]
Finally, the choice is to not expose the terminus_schedules API for the moment.
That is to say, we do not have **terminus_schedule** into the list of choice.
But the **layout** adaption is ready

**Pull request merged**